app/src/styles/_uswds-theme.scss

```diff
@@ -19,6 +19,9 @@ in the form $setting: value,
   $utilities-use-important: true,
 
   // Prettier defaults for typography
+  $theme-style-body-element: true,
   $theme-font-type-sans: "public-sans",
-  $theme-global-link-styles: true
+  $theme-global-link-styles: true,
+  $theme-global-paragraph-styles: true,
+  $theme-global-content-styles: true
```

Comment:
i have no strong preference but do we want to alphabetize this group?

Reply:
Sounds good to me, I can update this.